build-system/scripts/retry_10

```diff
@@ -1,7 +1,7 @@
 set -eu
 # Retries up to 10 times with 10 second intervals
 for i in $(seq 1 10); do
-    "$@" && return || sleep 10
```

Comment:
Not sure why this wasnt hit before but eventually got 'cant call return outside of function'